project/dexter-eclipse-ui/src/java/com/samsung/sec/dexter/eclipse/ui/action/FunctionMetricsDescroption.java

```diff
@@ -0,0 +1,103 @@
+package com.samsung.sec.dexter.eclipse.ui.action;
+
+import java.util.Iterator;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.eclipse.jface.action.IAction;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.viewers.ISelection;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.ui.IObjectActionDelegate;
+import org.eclipse.ui.IViewPart;
+import org.eclipse.ui.IWorkbenchPart;
+
+import com.samsung.sec.dexter.core.config.DexterConfig;
+import com.samsung.sec.dexter.core.exception.DexterRuntimeException;
+import com.samsung.sec.dexter.core.util.DexterClient;
+//import com.samsung.sec.dexter.eclipse.ui.notifier.NotificationType;
+//import com.samsung.sec.dexter.eclipse.ui.notifier.NotifierDialog;
+import com.samsung.sec.dexter.eclipse.ui.util.EclipseUtil;
+import com.samsung.sec.dexter.eclipse.ui.view.AnalysisLog;
+import com.samsung.sec.dexter.eclipse.ui.view.FunctionMetricsView;
+
+public class FunctionMetricsDescroption implements IObjectActionDelegate {
+	
+	static Logger logger = Logger.getLogger(FunctionMetricsDescroption.class);
+
+	private AnalysisLog analysisLog;
+	private String fileName;
+	private String modulePath;
+	
+	private List<String> functionList ;
+
+	private IWorkbenchPart part;
+	
+	public FunctionMetricsDescroption() {
+	}
+
+	@Override
+	public void run(IAction action) {
+		assert analysisLog != null;
+		
+		try{
+		IViewPart view = EclipseUtil.findView(FunctionMetricsView.ID);
+		final FunctionMetricsView functionMetricsView = (FunctionMetricsView) view;
+		
+		StringBuilder url = new StringBuilder();
+		url.append("http://").append(DexterClient.getInstance().getServerHost()).append(":") //$NON-NLS-1$ //$NON-NLS-2$
+		.append(DexterClient.getInstance().getServerPort()).append(DexterConfig.FUNCTION_METRICS_BASE)//$NON-NLS-1$
+		.append("?").append(DexterConfig.CODE_METRICS_FILE_NAME).append("=").append(fileName)//$NON-NLS-1$
+		.append("&").append(DexterConfig.CODE_METRICS_MODULE_PATH).append("=").append(modulePath);//$NON-NLS-1$
+		
+		for(int i=0;i<functionList.size(); i++){
```

Comment:
You can change this code like the below:

url.append("&")....append(functionList.get(0));
for(int i=1 ...){
    url.append(",").append(functionList.get(i));
}